spec/function.dd

```diff
@@ -561,6 +561,10 @@ $(H2 $(LNAME2 property-functions, Property Functions))
     $(LI $(D @property) functions cannot be overloaded with non-$(D @property) functions with the same name.)
     $(LI $(D @property) functions can only have zero, one or two parameters.)
     $(LI $(D @property) functions cannot have variadic parameters.)
+    $(LI Binary assignments can be used with $(D @property) functions.  If the getter does not return by `ref`,
+    the return value is not an aggregate that overloads the binary assignment operator,
+    and a setter is defined, binary assignment is equivalent to `setter(getter() @ value)`, where `@` is the
+    operation.)
```

Comment:
As mentioned in https://github.com/dlang/dmd/pull/7079, it should be specified whether the object is evaluated once or twice and whether it's copied or not (should be evaluated once and never copied). At best the spec would define the operation through a lowering to simpler constructs.